sdk/src/Services/DynamoDBv2/Custom/AWSConfigs.DynamoDB.cs

```diff
@@ -155,6 +155,17 @@ public partial class DynamoDBContextConfig
         /// </remarks>
         public MetadataCachingMode? MetadataCachingMode { get; set; }
 
+        /// <summary>
+        /// If true disables fetching table metadata automatically from DynamoDB. Table metadata must be 
+        /// defined by<see cref="DynamoDBAttribute"/> attributes and/or in <see cref = "AWSConfigsDynamoDB"/>.
```

Comment:
nit: Missing space between "by" and "DynamoDBAttribute"

Reply:
Fixed